src/pudl/transform/eia861.py

```diff
@@ -2261,12 +2259,7 @@ def core_eia861__yearly_reliability(
         # Drop duplicate entries for utilities 13027, 3408 and 9697
         .pipe(_drop_dupes, df_name="Reliability", subset=idx_cols)
         .pipe(_post_process)
-    )
-
-    transformed_r = (
-        pudl.metadata.classes.Package.from_resource_ids()
-        .get_resource("core_eia861__yearly_reliability")
-        .encode(transformed_r)
+        .pipe(PUDL_PACKAGE.encode)
```

Comment:
This is the only table that was getting encoded in the EIA-861.  I **could** put this step in the `_post_process()` function, but there may be columns in the (only partially processed) EIA-861 which match the FK relationship rules, but that haven't been cleaned up enough to encode yet.